klippy/kinematics/deltesian.py

```diff
@@ -0,0 +1,184 @@
+# Code for handling the kinematics of deltesian robots
+#
+# Copyright (C) 2022  Fabrice Gallet <tircown@gmail.com>
+#
+# This file may be distributed under the terms of the GNU GPLv3 license.
+import math, logging
+import stepper
+
+# Slow moves once the ratio of tower to XY movement exceeds SLOW_RATIO
+SLOW_RATIO = 3.
+
+# Minimum angle with the horizontal for the arm to not exceed - in degrees
+MIN_ANGLE = 5.
+
+class DeltesianKinematics:
+    def __init__(self, toolhead, config):
+        self.rails = [None] * 3
+        stepper_configs = [config.getsection('stepper_' + s)
+                                    for s in ['left', 'right', 'y']]
+        self.rails[0] = stepper.PrinterRail(
```

Comment:
This is not a requirement, of course, but why not call `LookupMultiRail` to support multi-stepper l/r rails?

Reply:
That's a good point. I take it into account.